pkgs/development/python-modules/imdbpy/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, lxml, sqlalchemy }:
+
+buildPythonPackage rec {
+  pname = "IMDbPY";
+  version = "2020.9.25";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p3j9j1jcgbw4626cvgpryhvczy9gzlg0laz6lflgq17m129gin2";
+  };
+
+  patches = [ ./sql_error.patch ]; # Already fixed in master, but not yet in the current release. This can be removed upon the next version update
```

Comment:
If it's an upstream commit we can just pull it with `fetchpatch` instead of pulling the patch into our git tree.

Reply:
That's what i initially tried as well, however when using [the entire commit](https://github.com/alberanid/imdbpy/commit/028da9fbb8737cbacfee122810157d1843bf88e0) the patch doesn't seem to apply properly:
```
applying patch /nix/store/l3mj5qsjl2vh5xn1wgv958pdlmfl3yg4-028da9fbb8737cbacfee122810157d1843bf88e0.patch
patching file CHANGELOG.txt
Hunk #1 FAILED at 1.
1 out of 1 hunk FAILED -- saving rejects to file CHANGELOG.txt.rej
patching file CREDITS.txt
patching file bin/imdbpy2sql.py
patching file imdb/parser/sql/__init__.py
builder for '/nix/store/pz2w0j0cb9qg6sk30qdnimfmbwfbjhf3-python3.8-IMDbPY-2020.9.25.drv' failed with exit code 1
error: build of '/nix/store/pz2w0j0cb9qg6sk30qdnimfmbwfbjhf3-python3.8-IMDbPY-2020.9.25.drv' failed
```

We only need the patch to `imdb/parser/sql/__init__.py`, so I've just manually copied that section over.

Reply:
That's fair, thanks!